docs/pages/eas-update/faq.mdx

```diff
@@ -18,6 +18,6 @@ EAS Update is a great way to quickly deliver improvements to the people who use
 
 ### Are Classic Updates still supported
 
-Expo is committed to supporting projects that use Classic Updates, which is the updates service available before December 2021. Apps that receive updates from the Classic Updates service will continue to work. With that, we are not planning on adding new features to Classic Updates other than fixing general issues and implementing security fixes as they arise.
+The Classic Updates service, which has been available before December 2021, is now deprecated. Users can no longer run `expo publish`, but existing apps will continue to receive Classic Updates that have already been published and are actively used.
 
-In the future, we plan to scale down Classic Updates as we transition to EAS Update. We will ensure every developer still using the service will have ample notice to upgrade to the modern Expo Updates format so that they can use another update service like their own servers or EAS Update.
+We recommend users transition to EAS Update or use a [self-hosted update service](https://docs.expo.dev/eas-update/custom-updates-server).
```

Comment:
```suggestion
We recommend transitioning to EAS Update or using a [self-hosted update service](/eas-update/custom-updates-server).
```

- "users" are implicit so let's avoid using that
- Also we can use relative links for internal docs page